### PR TITLE
Select service variants with "service up"

### PR DESCRIPTION
**Pull request by mtojek**
Issue: https://github.com/elastic/elastic-package/issues/94

This PR extends the `elastic-package service up` command to select right service variant to boot up. Service factory (for Docker Compose) expects variant information now.

How to test:

```
make build
cd test/packages/apache
elastic-package service up -v -s v2420
ctrl+c...

elastic-package service up -v -s v2443
```

### CODE REVIEW

File: cmd/service.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ func setupServiceCommand() *cobraext.Command {
 		RunE:  upCommandAction,
 	}
 	upCommand.Flags().StringP(cobraext.DataStreamFlagName, "d", "", cobraext.DataStreamFlagDescription)
+	upCommand.Flags().StringP(cobraext.VariantFlagName, "s", "", cobraext.VariantFlagDescription)
```
Comment (jsoriano) [inline cmd/service.go:29]: Nit. Why `s`? I think I would prefer to don't have an abbreviated form :slightly_smiling_face:

**Review by jsoriano — APPROVED** [review]
Nice to have this feature!

File: cmd/service.go (hunk 1 above)
Comment (mtojek) [reply cmd/service.go:29]: It's for `service variant`, it's because `-v` is reserved for `verbose`. I can drop the short form in this case :)

Comment (mtojek) [reply cmd/service.go]: Fixed

**Pull request merged**